src/libasr/codegen/wasm_to_x64.cpp

```diff
@@ -506,7 +500,7 @@ Result<int> wasm_to_x64(Vec<uint8_t> &wasm_bytes, Allocator &al,
     }
 
     //! Helpful for debugging
-    // std::cout << x64_visitor.m_a.get_asm() << std::endl;
+    std::cout << x64_visitor.m_a.get_asm() << std::endl;
```

Comment:
Probably want to remove.

Reply:
Sorry, I will be careful next time. I fixed it in [WASM_X64: Comment out asm text print](https://github.com/lcompilers/lpython/pull/1422/commits/c039edfe2026b74a9bf260e37b5a6e2eb86bc152).